webxrlayers-1.bs

```diff
@@ -797,6 +800,69 @@ interface XRMediaBinding {
 
 </pre>
 
+WebXR Device API Integration {#webxrintegration}
+============================
+
+XRRenderState changes {#xrrenderstatechanges}
+---------------------
+This module extends the {{XRRenderStateInit}} and {{XRRenderState}} interfaces with a new optional array {{XRRenderStateInit/layers}} containing
+instances of {{XRLayer}} or {{XRWebGLLayer}}.
+
+<pre class="idl">
+typedef (XRWebGLLayer or XRLayer) XRGenericLayer;
+
+dictionary XRRenderStateInit {
+  double depthNear;
+  double depthFar;
+  double inlineVerticalFieldOfView;
+  XRWebGLLayer? baseLayer;
+  FrozenArray&lt;XRGenericLayer&gt;? layers;
+};
+
+[SecureContext, Exposed=Window] interface XRRenderState {
+  readonly attribute double depthNear;
+  readonly attribute double depthFar;
+  readonly attribute double? inlineVerticalFieldOfView;
+  readonly attribute XRWebGLLayer? baseLayer;
+  readonly attribute FrozenArray&lt;XRGenericLayer&gt;? layers;
+};
```

Comment:
This can be a partial interface, and I think we should define it as such to highlight what this module is actually adding.

Reply:
Will do.